docs/HowToFAQ/FAQBoogie.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How do I run boogie manually on the Dafny output?
+---
+
+## Question
+
+Running Dafny on my program works, but trying to run Boogie on the bpl output from Dafny fails. What command-line arguments for Boogie am I missing?
+
+## Answer
+
+You can get a Boogie file from Dafny using the option `-print`. For example, the command `dafny -print:a.bpl HelloWorld.dfy` will produce a file `a.bpl`. 
```

Comment:
The question suggests that the asker already knows how to generate Boogie code.  I think the answer is excellent, so I think we should generalize the question.